examples/calculations/submit_geomopt.py

```diff
@@ -27,7 +27,7 @@ def geomopt(params: dict) -> None:
     structure = load_structure(params["struct"])
 
     # Select model to use
-    model = load_model(params["model"], params["architecture"])
+    model = load_model(params["model"], params["arch"])
```

Comment:
this needs to be fixed separately not in this.

Reply:
yea I know just I changed them cause I thought something was exploding because of these after modifying the singledata (obviously they were actually not the cause) and decided to leave the changes